dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -165,31 +163,11 @@ public static ZenodoDoiResult registerZenodoDOI(ApiClient zenodoClient, Workflow
 
         Deposit publishedDeposit = publishDepositOnZenodo(actionsApi, depositionID);
 
-        String conceptDoiUrl = publishedDeposit.getLinks().get("parent_doi");
-
-        String conceptDoi = extractDoiFromDoiUrl(conceptDoiUrl);
+        String conceptDoi = publishedDeposit.getLinks().get("conceptdoi");
```

Comment:
looking at https://zenodo.org/api/records/11093965, `conceptdoi` doesn't exist in the `links` array 

We previously had the code like this but @denis-yuen changed it in https://github.com/dockstore/dockstore/commit/021bf961dea88293145025e7947548fd6539cbd8 because that property doesn't exist anymore

I do see a top-level `conceptdoi` property, maybe you meant to reference that one?